draft-ietf-teep-architecture.md

```diff
@@ -260,7 +260,10 @@ The following terms are used:
     it is more common for the enterprise to own the device, and for any device
     user to have no or limited administration rights. In this case, the
     enterprise appoints a Device Administrator that is not the device
-    owner.
+    owner. A Device Owner may or may not have control over Untrusted Applications 
+    and TAs into the device depending on whether it is also a Device Administrator.
+    It is a Device Administrator's responsibility and privilege in general for
```

Comment:
what does "in general" mean?  Does that mean there are exceptions?   I think lines 246-255 already say this right before this paragraph so this PR should not be needed.

Reply:
The addition is to explicitly reiterate about an Owner's control over apps. I am good to defer this for now, I will close it without commit the PR. We can revisit it if Ben insists a change later.